src/app/core/cache/object-cache.effects.ts

```diff
@@ -18,10 +19,11 @@ export class ObjectCacheEffects {
    */
   @Effect() fixTimestampsOnRehydrate = this.actions$
     .pipe(ofType(StoreActionTypes.REHYDRATE),
-      map(() => new ResetObjectCacheTimestampsAction(new Date().getTime()))
+      map(() => new ResetObjectCacheTimestampsAction(new Date().getTime())),
+      tap(() => this.authorizationsService.invalidateAuthorizationsRequestCache())
```

Comment:
I wouldn't add this to the bottom of another, unrelated effect. I'd add a separate effect that listens to the same action, with `dispatch: false` especially for this. 

That way the effect can have a name that reflects what it does, it makes the code easier to understand, and to describe in typedocs.

Reply:
That way it can also be moved to auth.effects.ts which is likely a better fit

Reply:
done